pkg/gvks/supportedgvks/supportedgvks.go

```diff
@@ -144,3 +144,17 @@ func BasedOnDCL(serviceMetaLoader metadata.ServiceMetadataLoader) []schema.Group
 	}
 	return gvkList
 }
+
+func IsDirectByGVK(gvk schema.GroupVersionKind) bool {
```

Comment:
It's better to pass in gk here

Reply:
Can you please explain why? I thought GVK is more precise.